code/ui/alivehud/inventorywrapper/inventoryselection/InventorySelection.cs

```diff
@@ -175,28 +188,30 @@ private int NormalizeSlotIndex(int index, int maxIndex)
 
         private int GetKeyboardNumberPressed(InputBuilder input)
         {
-            if (input.Pressed(InputButton.Slot1)) return 1;
-            if (input.Pressed(InputButton.Slot2)) return 2;
-            if (input.Pressed(InputButton.Slot3)) return 3;
-            if (input.Pressed(InputButton.Slot4)) return 4;
-            if (input.Pressed(InputButton.Slot5)) return 5;
+            for (int i = 0; i < _slotInputButtons.Length; i++)
+            {
+                if (input.Pressed(_slotInputButtons[i]))
+                {
+                    return i;
+                }
+            }
```

Comment:
Could just create a map of `InputButton` -> `i` and then try and fetch the value from the map.